duet/views/mobile_ios_country.view.lkml

```diff
@@ -0,0 +1,368 @@
+            THEN
+              'MO'
+            WHEN
+              name = 'Eswatini'
+            THEN
+              'SZ'
+            WHEN
+              name = 'Laos'
+            THEN
+              'LA'
+            ELSE
+              code
+            END
+            AS country,
+            product_page_views,
+            first_time_installs,
+            installations_opt_in
+          FROM
+            apple_storefront_metrics
+          LEFT JOIN
+            `mozdata.static.country_codes_v1`
+          USING
+            (name)
+        ),
+        apple_countries AS (
+          SELECT DISTINCT
+            country
+          FROM
+            apple_country_metrics
+        ),
+        last_seen AS (
+          SELECT
+            first_seen_date AS submission_date,
+            coalesce(apple_countries.country, "OTHER") AS country,
+            count(DISTINCT client_id) AS first_seen,
+            count(
+              DISTINCT
+              CASE
+              WHEN
+                coalesce(BIT_COUNT(days_seen_bits), 0) >= 5
+              THEN
+                client_id
+              END
+            ) AS activated
+          FROM
+            `moz-fx-data-shared-prod.org_mozilla_ios_{% parameter.app_id %}.baseline_clients_last_seen`
+          LEFT JOIN
+            apple_countries
+          USING
+            (country)
+          CROSS JOIN
+            period
+          WHERE
+            submission_date
+              BETWEEN date_sub(current_date(), INTERVAL {% parameter.history_days %}*({% parameter.period_offset %} + 1) + 7 day)
+              AND date_sub(current_date(), INTERVAL {% parameter.history_days %}*{% parameter.period_offset %} day)
+            AND first_run_date
+            BETWEEN start_date
+            AND period.end_date
+          GROUP BY
+            1,
+            2
+        )
+      SELECT
+        submission_date,
+        country,
+        max(ios_store_updated) AS ios_store_updated,
+        max(latest_date) AS latest_date,
+        sum(product_page_views) AS product_page_views,
+        sum(first_time_installs) AS first_time_installs,
+        sum(installations_opt_in) AS installations_opt_in,
+        sum(first_seen) AS first_seen,
+        sum(activated) AS activated
+      FROM
+        apple_country_metrics
+      JOIN
+        last_seen
+      USING
+        (submission_date, country)
+      CROSS JOIN
+        period
+      WHERE
+        submission_date
+        BETWEEN start_date
+        AND end_date
+      GROUP BY
+        1,
+        2
+      ORDER BY
+        1,
+        2
+       ;;
+  }
+
+  parameter: app_id {
+    description: "The name of the application in the `org.mozilla` namespace."
+    type:  unquoted
+    default_value: "firefox"
+    allowed_value: {
+      label: "Firefox"
+      value: "firefox"
+    }
+    allowed_value: {
+      label: "Focus"
+      value: "focus"
+    }
+    allowed_value: {
+      label: "Klar (Focus)"
+      value: "klar"
+    }
+  }
```

Comment:
Can we do something like `app_name` here instead? `app_id` may be confusing.
```suggestion
  parameter: app_name {
    description: "The name of the application."
    type:  unquoted
    default_value: "firefox"
    allowed_value: {
      label: "Firefox for iOS"
      value: "firefox"
    }
    allowed_value: {
      label: "Focus for iOS"
      value: "focus"
    }
    allowed_value: {
      label: "Klar (Focus)"
      value: "klar"
    }
  }
```

Reply:
This is slightly inconsistent with the android view. Should I modify this there, too?

Reply:
I think that makes sense, just for greater clarity. I wouldn't expect everyone who's using the explores to know what an `app_id` is.